next.config.js

```diff
@@ -2,6 +2,9 @@
 const nextConfig = {
   reactStrictMode: true,
   output: "export",
+  compiler: {
+    styledComponents: true,
```

Comment:
For some reason, today I learn that useRef only works in client-side. hence, I had to add "use-client" in Dua.tsx which leads  to a problem of using nextjs fonts in Dua.tsx therefore I had to use this option in nextConfig to get rid of the exception.

Ref: https://stackoverflow.com/a/74917774